File: openapi/code_samples/nodejs/accounts_balance/code.js
var myHeaders = new Headers();
myHeaders.append("Accept", "application/com.reloadly.giftcards-v1+json");
myHeaders.append("Authorization", "Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX");
myHeaders.append("Content-Type", "application/json");

var requestOptions = {
  method: 'GET',
  headers: myHeaders,
  redirect: 'follow'
};

fetch("https://giftcards.reloadly.com/accounts/balance", requestOptions)
  .then(response => response.text())
  .then(result => console.log(result))
  .catch(error => console.log('error', error));

File: openapi/code_samples/python/accounts_balance/code.py
import requests

url = "https://giftcards.reloadly.com/accounts/balance"

payload={}
headers = {
  'Accept': 'application/com.reloadly.giftcards-v1+json',
  'Authorization': 'Bearer eyJraXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX',
  'Content-Type': 'application/json'
}

response = requests.request("GET", url, headers=headers, data=payload)

print(response.text)